external_parser/example_joiner.cc

```diff
@@ -244,37 +275,30 @@ int example_joiner::process_outcome(const v2::Event &event,
   outcome_event o_event;
   o_event.metadata = {"client_time_utc",
                       metadata.app_id() ? metadata.app_id()->str() : "",
-                      metadata.payload_type(),
-                      metadata.pass_probability(),
+                      metadata.payload_type(), metadata.pass_probability(),
                       metadata.encoding()};
   o_event.enqueued_time_utc = enqueued_time_utc;
 
-  auto outcome = v2::GetOutcomeEvent(event.payload()->data());
+  flatbuffers::DetachedBuffer detached_buffer;
```

Comment:
`detached_buffer` doesn't seem to be used

Reply:
true, removed